.ci/test.sh

```diff
@@ -29,7 +29,7 @@ if [[ "$TASK" == "cpp-tests" ]]; then
     exit 0
 fi
 
-conda create -q -y -n $CONDA_ENV python=$PYTHON_VERSION
+conda create -q -y -n $CONDA_ENV python=${PYTHON_VERSION}
```

Comment:
To simplify the diff by removing this unrelated line.
```suggestion
conda create -q -y -n $CONDA_ENV python=$PYTHON_VERSION
```